infra/app/service/outputs.tf

```
@@ -1,3 +1,8 @@
+output "load_balancer_endpoint" {
+  description = "The load balancer endpoint for the service."
+  value       = module.service.load_balancer_endpoint
+}
+
 output "service_endpoint" {
   description = "The public endpoint for the service."
   value       = module.service.public_endpoint
```

could we just expose the load balancer endpoint and not the underlying service endpoint? it was tech debt that i exposed the service directly. so i'm thinking something like

```suggestion
output "service_endpoint" {
  description = "The public endpoint for the service."
  value       = module.service.load_balancer_endpoint
```